packages/ts-morph/src/Project.ts

```diff
@@ -242,10 +242,10 @@ export class Project {
    * Gets a directory by the specified path or throws if it doesn't exist.
    * @param dirPath - Path to create the directory at.
    */
-  getDirectoryOrThrow(dirPath: string): Directory {
+  getDirectoryOrThrow(dirPath: string, message?: string): Directory {
```

Comment:
How hard would it be to make these `onErrorMessage?: string | () => string` so someone could provide a string that gets lazily created only when necessary?

Reply:
Good idea - I will take a look 👍

Reply:
Done